docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+available via Conda do not include the `NVIDIA CUDA Compiler (NVCC)`_ which is required in order 
+to build Horovod extensions for PyTorch, TensorFlow, or MXNet.
+
+What about the :code:`cudatoolkit-dev` package?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While there are :code:`cudatoolkit-dev` packages available from :code:`conda-forge` that do 
+include NVCC, we have had difficult getting these packages to consistently install properly. Some 
+of the available builds require manual intervention to accept license agreements making these 
+builds unsuitable for installing on remote systems (which is critical functionality). Other builds 
+seems to work on Ubuntu but not on other flavors of Linux.
+
+Despite this we would encourage you to try adding :code:`cudatoolkit-dev` to your 
+:code:`environment.yml` file and see what happens! The package is well maintained so perhaps it 
+will become more stable in the future.
+
+Use the :code:`nvcc_linux-64` meta-package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The most robust approach to obtain NVCC and still use Conda to manage all the other dependencies 
+is to install the NVIDIA CUDA Toolkit on your system and then install a meta-package 
+`nvcc_linux-64`_ from conda-forge which configures your Conda environment to use the NVCC 
+installed on the system together with the other CUDA Toolkit components installed inside the Conda 
+environment.
+
+The :code:`environment.yml` file
+--------------------------------
+
+We prefer to specify as many dependencies as possible in the Conda :code:`environment.yml` file 
+and only specify dependencies in :code:`requirements.txt` for install via :code:`pip` that are not 
+available via Conda channels. Check the Horovod `installation guide`_ for details of required 
+dependencies.
+
+Channel Priority
+^^^^^^^^^^^^^^^^
+
+Use the recommended channel priorities. Note that :code:`conda-forge` has priority over 
+:code:`defaults` and :code:`pytorch` has priority over :code:`conda-forge`. ::
+
+    name: null
+
+    channels:
+    - pytorch
+    - conda-forge
+    - defaults
+
+Dependencies
+^^^^^^^^^^^^
+
+There are a few things worth noting about the dependencies.
+
+1. Even though you have installed the NVIDIA CUDA Toolkit manually you should still use Conda to 
+   manage the other required CUDA components such as :code:`cudnn` and :code:`nccl` (and the 
+   optional :code:`cupti`).
+2. Use two meta-packages, :code:`cxx-compiler` and :code:`nvcc_linux-64`, to make sure that 
+   suitable C, and C++ compilers are installed and that the resulting Conda environment is aware 
+   of the manually installed CUDA Toolkit.
+3. Horovod requires some controller library to coordinate work between the various Horovod 
+   processes. Typically this will be some MPI implementation such as `OpenMPI`_. However, rather 
+   than specifying the :code:`openmpi` package directly you should instead opt for `mpi4py`_ Conda 
```

Comment:
Add a comma after "directly"
I recommend using double backquotes (``) for code highlighting

Reply:
Done